packages/terra-overlay/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Unreleased
 
 3.38.0 - (January 7, 2020)
 ------------------
+### Changed
+* Updated screenshot
```

Comment:
This entry is in a released section. Was the updated screenshot diff removed?

Reply:
this was cleaned up in this pr: https://github.com/cerner/terra-core/pull/2791

it was a change that resulted from reverting the axe-xore upgrade in toolki.